packages/raystack/components/progress/progress.module.css

```
@@ -53,7 +55,10 @@
 
 .circularSvg {
   --rs-progress-track-size: var(--rs-space-2);
-  --rs-progress-radius: calc((var(--rs-space-14) - var(--rs-progress-track-size) * 2) / 2);
+  --rs-progress-radius: calc(
+    (var(--rs-space-14) - var(--rs-progress-track-size) * 2) /
```

Stylelint (17.14.0)

[error] Unexpected newline after "/" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)